### PR TITLE
Runtime 10.8 + 9.10.2 + 9.13 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -60,6 +60,16 @@ Refer to the [Airflow release notes](https://airflow.apache.org/docs/apache-airf
...
+## Astro Runtime 10.8.0
+
+- Release date: April, 18 2024
+- Airflow version: 2.8.4
+
+### Additional improvements
+
+- Updated `sqlparse` to `0.5.0`.
+- Upgraded [Gunicorn](https://gunicorn.org/) to `22.0.0`.
+
```
Comment (utkarsharma2) [inline astro/runtime-release-notes.md:72]: We need to include the changes for PR - https://github.com/astronomer/astro-runtime/pull/1426

File: astro/runtime-release-notes.md [hunk 2]
```diff
@@ -188,6 +198,20 @@ For more information about the major changes in this release, see the [Airflow b
...
+## Astro Runtime 9.13.0
+
+- Release date: April, 18 2024
+- Airflow version: 2.7.3
+
+### Additional improvements
+
+- Upgraded [Gunicorn](https://gunicorn.org/) to `22.0.0`.
+- Updated the version of `sqlparse` to `0.5.0`.
+
```
Comment (utkarsharma2) [inline astro/runtime-release-notes.md:210]: We need to include the changes for PR - https://github.com/astronomer/astro-runtime/pull/1426

**Review by utkarsharma2 — APPROVED** [review]

File: astro/runtime-release-notes.md [hunk 3]
```diff
@@ -60,6 +60,16 @@ Refer to the [Airflow release notes](https://airflow.apache.org/docs/apache-airf
...
 - In the Python task decorator, you can only have `None` as the default parameter for context parameters.
 
+## Astro Runtime 10.8.0
+
+- Release date: April, 18 2024
+- Airflow version: 2.8.4
+
+### Additional improvements
+
+- Updated `sqlparse` to `0.5.0`.
```
Comment (jwitz) [inline astro/runtime-release-notes.md:70]:
```suggestion
- Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release. 
- Updated `sqlparse` to `0.5.0`.
```

File: astro/runtime-release-notes.md (hunk 2 above)
Comment (jwitz) [inline astro/runtime-release-notes.md:210]:
```suggestion
- Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release. 

```

File: astro/runtime-release-notes.md [hunk 4]
```diff
@@ -188,6 +199,21 @@ For more information about the major changes in this release, see the [Airflow b
...
+## Astro Runtime 9.13.0
+
+- Release date: April, 18 2024
+- Airflow version: 2.7.3
+
+### Additional improvements
+
+- Upgraded [Gunicorn](https://gunicorn.org/) to `22.0.0`.
+- Updated the version of `sqlparse` to `0.5.0`.
+- Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release.
```
Comment (jwitz) [inline astro/runtime-release-notes.md:211]:
```suggestion
- Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release. 
- Upgraded [Gunicorn](https://gunicorn.org/) to `22.0.0`.
- Updated the version of `sqlparse` to `0.5.0`.
```

File: astro/runtime-release-notes.md [hunk 5]
```diff
@@ -188,6 +199,21 @@ For more information about the major changes in this release, see the [Airflow b
...
 - Fixed an issue where Astro audit logs listed a user's name as `User` for trigger events instead of their IDs.
 
+## Astro Runtime 9.13.0
+
+- Release date: April, 18 2024
+- Airflow version: 2.7.3
+
+### Additional improvements
+
+- Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release.
```
Comment (jwitz) [inline astro/runtime-release-notes.md:209]:
```suggestion
- Added functionality for using plugins to generate custom menu items in the Airflow UI. This feature will be fully available on Astro in a future release. 
```

File: astro/runtime-release-notes.md [hunk 6]
```diff
@@ -60,6 +60,17 @@ Refer to the [Airflow release notes](https://airflow.apache.org/docs/apache-airf
...
 - In the Python task decorator, you can only have `None` as the default parameter for context parameters.
 
+## Astro Runtime 10.8.0
+
+- Release date: April, 18 2024
+- Airflow version: 2.8.4
+
+### Additional improvements
+
+- Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release.
```
Comment (jwitz) [inline astro/runtime-release-notes.md:70]:
```suggestion
- Added functionality for using plugins to generate custom menu items in the Airflow UI. This feature will be fully available on Astro in a future release. 
```

**Review by lzdanski — APPROVED** [review]
After adding suggested changes, LGTM

**Pull request merged**